modules/apps/27-interchain-accounts/host/types/msgs.go

```diff
@@ -0,0 +1,25 @@
+package types
+
+import sdk "github.com/cosmos/cosmos-sdk/types"
+
+var _ sdk.Msg = (*MsgUpdateParams)(nil)
+
+// ValidateBasic implements sdk.Msg
+func (msg MsgUpdateParams) ValidateBasic() error {
+	_, err := sdk.AccAddressFromBech32(msg.Authority)
+	if err != nil {
+		return err
```

Comment:
seeing something similar in the transfer PR (different error message used there though)

```suggestion
		return errorsmod.Wrapf(ibcerrors.ErrInvalidAddress, "string could not be parsed as address: %v", err)
```

Reply:
done